### DISCUSSION TITLE
Legacy utilities missing from latest version of IM

### DISCUSSION THREAD

**Discussion by clintpike** [ImageMagick/ImageMagick#7452] [Help]
### ImageMagick version

7.1.1-34

### Operating system, version and so on

Windows

### Description

Hey, just downloaded ImageMagick-7.1.1-34-Q16-HDRI-x64-dll.exe - there was no option to enter an installation location and install legacy utilities was ticked by default. Anyway it installed to C:\Program Files\ImageMagick but the legacy utilities are missing e.g convert.exe.
Is this a bug with the release? How can I access an old version with the legacy tools?
Thanks

**Comment by dlemstra** [comment]
Only the `convert.exe` utility should no longer be included, the rest of the tools should still be available. You should stop using `convert.exe` and switch to `magick.exe`.

**Reply by clintpike** [reply]
Yeah sorry on closer inspection, it is just convert.exe. Our app is hard-coded to use convert.exe - not a quick fix. Do you know the latest version that contains convert.exe and where to download it, please? The old versions seem to just be the files and not the legacy tools. Thanks

**Reply by dlemstra** [reply]
This is the first release where we removed the `convert.exe` executable. We don't keep an archive of older versions so you cannot download it somewhere.

**Reply by fmw42** [reply]
You could try https://launchpad.net/imagemagick/+download

**Comment by snibgo** [comment]
You could create a symbolic link. For example, in Windows, in the directory that contains `magick.exe`:
```
mklink /H convert.exe magick.exe
```
Then a `convert` command will use the (very old) v6 syntax.

I do not recommend this as a long-term solution. For that, I suggest updating your app. This should include modifying commands, if required, to use the much better v7 `magick` syntax.

**Reply by dlemstra** [reply]
As Snibgo suggest this could be a temporary work around. But this will at some point break. And that might happen sooner than you think so my advice would be to stop using the `convert` syntax and switch to `magick`.
